src/components/task-plan/homework/add-exercises.cjsx

```diff
@@ -81,8 +82,10 @@ AddExercises = React.createClass
     TaskPlanStore.hasExercise(@props.planId, exercise.id)
 
   setCurrentSection: (currentSection) ->
+    @scrollToSelector("[data-section='#{currentSection}']")
```

Comment:
yes, good idea.  Just updated `src/components/exercises/cards.cjsx` to note the importance of the `data-section` attribute